db/seeds.rb

```diff
@@ -9,15 +9,11 @@
 
 john = User.create(username: 'user',password: 'password', first_name: 'John', last_name: 'Doe', email: 'example@example.com', role: 1)
 
-category1 = Category.create(name: 'Coffee', image: "fa fa-coffee")
-category2 = Category.create(name: 'Brewing', image: "fa fa-tint")
-category3 = Category.create(name: 'Merchandise', image: "fa fa-gratipay")
-category4 = Category.create(name: 'Chow', image: "fa fa-cutlery")
-category5 = Category.create(name: 'Shop All', image: "fa fa-archive")
-
-coffee = Category.create!(name: 'Coffee')
-brewing = Category.create!(name: 'Brewing')
-merchandise = Category.create!(name: 'Merchandise')
+coffee = Category.create(name: 'Coffee', image: "fa fa-coffee")
+brewing = Category.create(name: 'Brewing', image: "fa fa-tint")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.